setup.py

```diff
@@ -0,0 +1,9 @@
+from setuptools import setup, find_packages
+
+setup(
+    name="tfdata_image_loader",
+    version="0.1",
+    packages=find_packages(),
+    description="A custom module for loading image data for Image Classification problem.",
```

Comment:
I'd suggest using single quote over double quote - it's more readable and more common. But just a nitpick :)